src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -969,28 +967,28 @@ function toggleMcqGeneratedOptions(checkbox, questionNumber) {
     var idSuffix = getQuestionIdSuffix(questionNumber);
 
     if (checkbox.checked) {
-        $('#mcqChoiceTable' + idSuffix).find('input[type=text]').prop('disabled', true)
+        $('#mcqChoiceTable' + idSuffix).find('input[type=text]').prop('disabled', true);
         $('#mcqChoiceTable' + idSuffix).hide();
         $('#mcqGenerateForSelect' + idSuffix).prop('disabled', false);
-        $("#mcqOtherOptionFlag" + idSuffix).parent().hide();
+        $("#mcqOtherOptionFlag" + idSuffix).closest('.checkbox').hide();
         $('#generatedOptions' + idSuffix).attr('value',
                                                $('#mcqGenerateForSelect' + idSuffix).prop('value'));
     } else {
         $('#mcqChoiceTable' + idSuffix).find('input[type=text]').prop('disabled', false);
         $('#mcqChoiceTable' + idSuffix).show();
         $('#mcqGenerateForSelect' + idSuffix).prop('disabled', true);
-        $("#mcqOtherOptionFlag" + idSuffix).parent().show();
+        $("#mcqOtherOptionFlag" + idSuffix).closest('.checkbox').show();
         $('#generatedOptions' + idSuffix).attr('value', 'NONE');
     }
 }
 
 function toggleMcqOtherOptionEnabled(checkbox, questionNumber) {
-	idOfQuestion = '#form_editquestion-' + questionNumber;
-	idSuffix = getQuestionIdSuffix(questionNumber);
+    idOfQuestion = '#form_editquestion-' + questionNumber;
+    idSuffix = getQuestionIdSuffix(questionNumber);
 
-	if($(idOfQuestion).attr('editStatus') === "hasResponses") {
-		$(idOfQuestion).attr('editStatus', "mustDeleteResponses");
-	}
+    if($(idOfQuestion).attr('editStatus') === "hasResponses") {
+        $(idOfQuestion).attr('editStatus', "mustDeleteResponses");
+    }
```

Comment:
please open d.firstTimer issue for this kind of stuff. It is not related to this pr.